HCTextField/Source/HCImages.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Don't make the image part of the textfield source code. Let the application pass the checkmark image to you. Add a optional property of checkmark. The application can choose which checkmark image to use.